src/diffusers/schedulers/__init__.py

```diff
@@ -27,6 +27,7 @@
     from .scheduling_ddpm import DDPMScheduler
     from .scheduling_deis_multistep import DEISMultistepScheduler
     from .scheduling_dpmsolver_multistep import DPMSolverMultistepScheduler
+    from .scheduling_dpmsolver_sde import DPMSolverSDEScheduler
```

Comment:
we will need this under a `is_torchsde_available()` similarily to: https://github.com/huggingface/diffusers/blob/86ecd4b795f865b5b615b8c54991c177bb3dbef5/src/diffusers/__init__.py#L149